app/src/main/java/org/mozilla/tv/firefox/pocket/PocketViewModel.kt

```diff
@@ -50,6 +59,6 @@ class PocketViewModel(pocketRepo: PocketRepo) : ViewModel() {
         url = "https://www.mozilla.org/en-US/",
         thumbnailURL = "https://blog.mozilla.org/firefox/files/2017/12/Screen-Shot-2017-12-18-at-2.39.25-PM.png,",
         popularitySortId = it
-        )
+    )
```

Comment:
nit: maybe `) }`? We open the braces on the same lines so it's intuitive to close on the same line, right?